### PR TITLE
feat: add config insights for individual configs

### PR DESCRIPTION
**Pull request by gitstart**
Closes [#911](https://github.com/flanksource/flanksource-ui/issues/911)

[Loom Video](https://www.loom.com/share/cbee5ae644ee455cacdffba284cc3d6a)

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

**Comment by gitstart** [comment]
@moshloop this PR is ready for review

**Comment by gitstart** [comment]
@moshloop this PR is ready for review

File: src/pages/config/InsightsDetailsChangesPage.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import { useParams, useSearchParams } from "react-router-dom";
```
Comment (mainawycliffe) [inline src/pages/config/InsightsDetailsChangesPage.tsx:1]: Let's rename this to something like ConfigDetailsInsights instead; adding changes to the name is a bit confusing

**Review by mainawycliffe — CHANGES_REQUESTED** [review]

**Comment by mainawycliffe** [comment]
We also need a Page Title for the page as well, similar to Config Changes in Style.

**Comment by gitstart** [comment]
> We also need a Page Title for the page as well, similar to Config Changes in Style.

Thanks for the feedback @mainawycliffe. For the page title should look like this, right?

Screenshot 2023-03-27 at 08 52 34

File: src/pages/config/ConfigDetailsInsights.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      typeof itemError === "symbol"
+        ? itemError
+        : itemError?.message ?? "Something went wrong";
+
+    return <InfoMessage message={errorMessage} />;
+  }
+
+  return (
+    <>
+      <Head prefix={configItem ? `Config Insights - ${configItem.name}` : ""} />
```
Comment (mainawycliffe) [inline src/pages/config/ConfigDetailsInsights.tsx:41]: May be just default to Config Insights when config insight is not available

Comment (gitstart) [reply src/pages/config/ConfigDetailsInsights.tsx]: Done @mainawycliffe

**Review by mainawycliffe — APPROVED** [review]
LGTM

**Comment by moshloop** [comment]
@mainawycliffe  why do we need this ? isn't it a duplicate of the side panel ?

**Pull request closed without merging**